allennlp/modules/token_embedders/pretrained_transformer_embedder.py

```diff
@@ -1,41 +1,132 @@
+import math
+
 from overrides import overrides
 from transformers.modeling_auto import AutoModel
+from transformers.tokenization_auto import AutoTokenizer
 import torch
+import torch.nn.functional as F
 
+from allennlp.data.token_indexers import PretrainedTransformerIndexer
 from allennlp.modules.token_embedders.token_embedder import TokenEmbedder
 
 
 @TokenEmbedder.register("pretrained_transformer")
 class PretrainedTransformerEmbedder(TokenEmbedder):
     """
     Uses a pretrained model from `transformers` as a `TokenEmbedder`.
+
+    # Parameters
+
+    model_name : `str`
+        The name of the `transformers` model to use. Should be the same as the corresponding
+        `PretrainedTransformerIndexer`.
+    max_len : `int`, optional (default = -1)
```

Comment:
Full name.  Also, `None` might be a slightly better default value for all of these parameters - it's hard to do the wrong thing on accident that way, because the default value isn't comparable to other ints.